packages/mobile/src/screens/app-drawer-screen/left-nav-drawer/LeftNavDrawer.tsx

```diff
@@ -51,15 +70,45 @@ type AccountDrawerProps = DrawerContentComponentProps & {
 }
 
 const useStyles = makeStyles(({ spacing, palette }) => ({
-  notificationBubble: {
-    height: spacing(3),
-    width: spacing(3),
-    borderRadius: spacing(3),
-    backgroundColor: palette.secondary,
-    marginLeft: spacing(2)
+  tokens: {
+    ...flexRowCentered(),
+    padding: spacing(0.5),
+    marginLeft: spacing(4),
+    borderWidth: 1,
+    borderColor: palette.neutralLight8,
+    backgroundColor: palette.neutralLight10,
+    borderRadius: spacing(25),
+    gap: spacing(2)
+  },
+  tokenAmount: {
+    paddingRight: spacing(1.5),
```

Comment:
hmm should probably call out uneven spacing values to julian in the figma

Reply:
it's not necessarily uneven though